src/widget/form/profileform.cpp

```diff
@@ -416,3 +431,83 @@ void ProfileForm::retranslateUi()
     // We have to add the toxId tooltip here and not in the .ui or Qt won't know how to translate it dynamically
     toxId->setToolTip(tr("This bunch of characters tells other Tox clients how to contact you.\nShare it with your friends to communicate."));
 }
+
+void ProfileForm::showRegisterToxme()
+{
+    bodyUI->toxmeUsername->setText("");
+    bodyUI->toxmeBio->setText("");
+    bodyUI->toxmePrivacy->setChecked(false);
+
+    bodyUI->toxmeRegisterButton->show();
+    bodyUI->toxmeUpdateButton->hide();
+    bodyUI->toxmePassword->hide();
+    bodyUI->toxmePasswordLabel->hide();
+}
+
+void ProfileForm::showExistenToxme()
```

Comment:
Sorry for my bad english :(
What about rename `showRegisterToxme()` to `showRegisteredToxme()` or something like that?